.github/workflows/dependabot-update.yml

```diff
@@ -7,9 +7,12 @@
 name: Dependabot updates
 
 on:
-  schedule:
+ push:
+      branches:
+        - dependabot-issue-npm
```

Comment:
```suggestion
# In case we need to change, uncomment out and change name to branch you are fixing on.
# Commits will make action trigger. 
# Remember to comment out after work done.
# push:
#     branches:
#      - dependabot-issue-npm
```